package.json

```diff
@@ -79,10 +81,24 @@
     "lint-staged": "^9.4.2",
     "lodash": "^4.17.21",
     "plop": "^2.5.3",
+    "css-to-react-native-transform": "^2.0.0",
     "postcss": "^7.0.36",
+    "postcss-calc": "7.0.2",
+    "postcss-cli": "^6.1.2",
```

Comment:
I no longer think these changes are needed. I am going to verify and undo these changes if they aren't needed.